core/overlays/test/configmaps.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
   name: solvers
 data:
   solvers: |
-    solver-rhel-8-py36
```

Comment:
We could keep the old solver for py36 here. Test environment is smaller so we don't need to trigger ingestion there.

Reply:
ack, thanks for pointing it out.